content/en/benchmark.md

```diff
@@ -0,0 +1,128 @@
+# NumPy Benchmarking
+
+
+![output-modify-2-new](https://user-images.githubusercontent.com/62256509/127653679-5151e22d-d141-4621-abb9-b3e6dd7988b7.jpg)
+
+
+## Implemented Algorithms
+
+Till this point 5 algorithms are implemented as mentioned bellow:
+
+### NumPy
+
+NumPy is a most fundamental package for scientific computing in Python. The major advantage of using NumPy is these gives same computational speed as C and Fortran. Let's see our inplementation details:
```

Comment:
I'm thinking what should come next is indeed a comparison with C/C++ and Fortran, like this sentence says?

The accelerators can be a second part of the story.

Reply:
The text should explain that this is a numerical simulation of the N-body problem and giv ethe constraints of the problem

C/C++ and Fortran code simulations can be found in the original repo https://github.com/paugier/nbabel/